string message
float32 steering
float32 throttle
bool button1
bool button2


